### PR TITLE
Update block Beacon APIs to Capella

### PR DESCRIPTION
**Pull request by rkapka**
**What type of PR is this?**

Feature

**What does this PR do? Why is it needed?**

Updates the following endpoints to allow Capella blocks:
- https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/getBlockV2 (SSZ only because regular was done in the previous PR)
- https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/getBlindedBlock (regular+SSZ)
- https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/publishBlock
- https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/publishBlindedBlock

Big thanks to @potuz for implementing endpoints on `capella`, I mostly refactored and added tests.

### CODE REVIEW

File: proto/eth/v2/BUILD.bazel [hunk 1]
```diff
@@ -43,6 +43,8 @@ ssz_gen_marshal(
         "SignedBlindedBeaconBlockBellatrix",
         "SignedBeaconBlockCapella",
         "SignedBlindedBeaconBlockCapella",
+        "BlsToExecutionChange",
+		"SignedBlsToExecutionChange",
```
Comment (terencechain) [inline proto/eth/v2/BUILD.bazel:47]: Can we format this?

File: proto/migration/v1alpha1_to_v2.go [hunk 2]
```diff
@@ -263,15 +318,197 @@ func V1Alpha1BeaconBlockBellatrixToV2Blinded(v1alpha1Block *ethpbalpha.BeaconBlo
 	return v2Block, nil
 }
 
-// V1Alpha1BeaconBlockBlindedCapellaToV2Blinded converts a v1alpha1 blinded Capella beacon block to a v2 blinded Capella block.
-func V1Alpha1BeaconBlockBlindedCapellaToV2Blinded(v1alpha1Block *ethpbalpha.BlindedBeaconBlockCapella) (*ethpbv2.BlindedBeaconBlockCapella, error) {
-	marshaledBlk, err := proto.Marshal(v1alpha1Block)
+// V1Alpha1BeaconBlockCapellaToV2Blinded converts a v1alpha1 Capella beacon block to a v2
+// blinded Capella block.
+func V1Alpha1BeaconBlockCapellaToV2Blinded(v1alpha1Block *ethpbalpha.BeaconBlockCapella) (*ethpbv2.BlindedBeaconBlockCapella, error) {
```
Comment (terencechain) [inline proto/migration/v1alpha1_to_v2.go:323]: Can we please add a unit test for this? It looks like an important function and we wanna make sure the new fields are not neglected

Comment (rkapka) [reply proto/migration/v1alpha1_to_v2.go:323]: This function is not needed in this PR. I will remove it.

**Comment by terencechain** [comment]
Lgtm! do we want a second eye for this? @potuz @nisdas or @james-prysm ?

**Review by saolyn — COMMENTED** [review]
lgtm too

**Review by saolyn — APPROVED** [review]

**Review by james-prysm — APPROVED** [review]
Lgtm

**Pull request merged**